src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,102 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    private final String PATH_FILE_RESULTS = "src/main/com/example009/radek/v3/calculator-results/";
+
+    void storeResultsIntoFile(List<Double> inputList) {
+
+        String name = getUserInput("Please insert file name where you want to save results: ");
+        saveResultsIntoFile(name, inputList);
+    }
+
+    void loadResultsFromFile() throws IOException {
```

Comment:
nacitani funguje, ale chtelo by to odladit ... udelejme to takhle:
zapnu program ... 1) Do you want load last result? (name of file / -)
2) Enteer new name of file for storing of result or press ENTER for for same which is used for loading

Reply:
Tady bych potřeboval trochu upřesnění: zapnu aplikaci, ta se mě zeptá jestli chci načíst soubor s výsledky, nebo jak chci pojmenovat nový soubor, takže uživatel jednou zadá 1 nebo 2 podle toho co si vybere a následně vypíše název souboru, ale nechápu ten enter, kdyby chtěl použít stejný název jako u bodu 1 ? Jakože by chtěl přepsat soubor s minulými výsledky ?

Reply:
ok, udelejme to nasledovne 
1) zeptas se zda chces nacist ulozene vysledky yes/no ... tato, hlaska ti naskoci jen pokud bude uz nejaky soubor existovat, budes je mit ulozene ve stjene slozce, tak vzdy jen vypises vse co v ni bude, predpokladejme, ze je tam vzdy vse validni
2) pokud existovat budou, tak vpyises jejich list a pred kazdy das cislo a hlasku ... enter number of file which you wnat to open ... takze budes mit ...'
1 - file1.txt
2 - petr.txt .. atd 
3) v kazde fazi bude moznost zadat exit

Reply:
![image](https://user-images.githubusercontent.com/5745308/200676158-a1b60911-1cc3-40f6-8f3d-5040bae6a673.png)